packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -170,7 +181,11 @@ export class Relayer {
 
     // b) Balance is large enough to instant relay. Deposit is in any state except finalized (i.e can be slow relayed
```

Comment:
```suggestion
    // b) Balance is large enough to instant relay and the relay does not have an instant relayer. Deposit is in any state except finalized (i.e can be slow relayed
```

Reply:
is this always the case? what if the deposit was relayed, sped up, and disputed. can the bot not then consider this to re-speedup?

Reply:
If the second relay has different params, then there won't be an instant relayer. If the relay has the same params, there will be an instant relayer. WDYT?